src/components/clusterConfiguration/BaremetalInventory.tsx

```diff
@@ -62,6 +63,11 @@ const BaremetalInventory: React.FC<{ cluster: Cluster }> = ({ cluster }) => {
         ) : (
           <HostRequirements ContentComponent={HostRequirementsContent} />
         )}
+        <CheckboxField
+          name="useExtraDisksForLocalStorage"
+          label="Use extra disks for local storage."
+          helperText="Non-boot disks will be usable by workloads for persistent storage."
+        />
```

Comment:
If we decide to keep the field here (it is temporary in any case as it will move to 'Storage' wizard step once it is introduced, this will fix the styling:
```suggestion
        />
        <Text />    
```
alternatively apply `margin-bottom: var(--pf-c-content--MarginBottom);` to the checkbox

Reply:
fixed fab4c9dee469f4df5314c27f09ec026454f118bd